### PR TITLE
feat(eviction): support rootfs eviction

### PR DESCRIPTION
**Pull request by nightmeng**
#### What type of PR is this?
Features

#### What this PR does / why we need it:
Support rootfs eviction.

### CODE REVIEW

**Review by waynepeking348 — APPROVED** [review]

**Review by zzzzhhb — DISMISSED** [review]

File: pkg/agent/evictionmanager/plugin/rootfs/rootfs_pressure.go [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+	return
+}
+
+func (r *PodRootfsPressureEvictionPlugin) ThresholdMet(_ context.Context) (*pluginapi.ThresholdMetResponse, error) {
+	resp := &pluginapi.ThresholdMetResponse{
+		MetType:       pluginapi.ThresholdMetType_NOT_MET,
+		EvictionScope: EvictionScopeSystemRootfs,
+	}
+
+	config := r.dynamicConfig.GetDynamicConfiguration().RootfsPressureEvictionConfiguration
```
Comment (luomingmeng) [inline pkg/agent/evictionmanager/plugin/rootfs/rootfs_pressure.go:99]: variable 'config' collides with imported package name

File: pkg/metaserver/agent/metric/helper/expire.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+*/
+
+package helper
+
+import (
+	"github.com/kubewharf/katalyst-core/pkg/metaserver/agent/metric"
+	utilmetric "github.com/kubewharf/katalyst-core/pkg/util/metric"
+)
+
+func IgnoreMetricDataExpired(metricData utilmetric.MetricData, err error) (utilmetric.MetricData, error) {
```
Comment (luomingmeng) [inline pkg/metaserver/agent/metric/helper/expire.go:24]: IgnoreMetricDataExpired is unused?

File: pkg/agent/evictionmanager/plugin/memory/helper.go [hunk 3]
```diff
@@ -74,13 +75,15 @@ const (
 type EvictionHelper struct {
 	metaServer         *metaserver.MetaServer
 	emitter            metrics.MetricEmitter
+	dynamicConfig      *dynamic.DynamicAgentConfiguration
```
[the remark this answers is not in the document]
Comment (nightmeng) [reply pkg/agent/evictionmanager/plugin/memory/helper.go:78]: done

File: pkg/agent/evictionmanager/plugin/rootfs/rootfs_pressure.go (hunk 1 above)
Comment (nightmeng) [reply pkg/agent/evictionmanager/plugin/rootfs/rootfs_pressure.go:99]: done

File: pkg/metaserver/agent/metric/helper/expire.go (hunk 2 above)
Comment (nightmeng) [reply pkg/metaserver/agent/metric/helper/expire.go:24]: yes, it is unused now.

**Review by sun-yuliang — APPROVED** [review]

**Review by csfldf — APPROVED** [review]

**Review by waynepeking348 — APPROVED** [review]

**Review by luomingmeng — APPROVED** [review]

**Review by zzzzhhb — APPROVED** [review]

**Pull request merged**